metricbeat/docs/modules/graphite/server.asciidoc

```diff
@@ -3,6 +3,9 @@ This file is generated! See scripts/docs_collector.py
 ////
 
 [[metricbeat-metricset-graphite-server]]
+=== Graphite server metricset
+
+beta[]
```

Comment:
To be safe and avoid asciidoc build problems in the future, I would add an empty line after each beta/experimental tag. (Sometimes people delete empty trailing/leading lines in referenced files or blocks of text, and this results in asciidoc errors.)

Reply:
added